src/SourceEuler.cpp

```diff
@@ -619,7 +619,7 @@ void calculate_qplus(t_data &data)
 			viscous_heating(data);
     }
     if (parameters::heating_star_enabled) {
-		if (!parameters::cooling_surface_enabled) {
+		if (!(parameters::cooling_surface_enabled | parameters::cooling_scurve_enabled)) {
```

Comment:
Shouldn't that be `||` i.e. the logical `or` operator?

Reply:
Yes it should.